pkg/distributor/ha_tracker.go

```diff
@@ -524,11 +526,14 @@ func (e replicasNotMatchError) IsOperationAborted() bool {
 }
 
 type tooManyClustersError struct {
-	limit int
+	limit  int
+	actual int
 }
 
 func (e tooManyClustersError) Error() string {
-	return fmt.Sprintf("too many HA clusters (limit: %d)", e.limit)
+	return globalerror.TooManyHAClusters.MessageWithLimitConfig(
+		validation.HATrackerMaxClustersFlag,
+		fmt.Sprintf("too many HA clusters (limit: %d, actual: %d)", e.limit, e.actual))
```

Comment:
The reason why we didn't have the actual number here is because it will always be equal to limit (or limit+1, depending how limit is implemented). Since we stop ingesting new HA clusters once you hit the limit, it will never go beyond the limit, so visualizating the actual number is not super correct (doesn't tell the real number of HA clusters of a tenant). I would rollback this change.